File: 3100/golang/3110. Score of a String.go
func scoreOfString(s string) int {
	ans := 0
	for i := 1; i < len(s); i++ {
		ans += abs(int(s[i-1]) - int(s[i]))
	}
	return ans
}

func abs(i int) int {
	if i < 0 {
		return -i
	}
	return i
}

File: 3100/rust/3110. Score of a String.rs
impl Solution {
    pub fn score_of_string(s: String) -> i32 {
        let mut result=0;
        let mut value=vec![];
        let mut my_string=s.chars();
        while let Some(ch) = my_string.next() {
            println!("{}",ch);
            if ch.is_ascii() {
                value.push(ch as i32)
            }
        }
        for i in 1..value.len() {
            result+=(value[i-1]-value[i]).abs();
        }
        result
    }
}